### ISSUE TITLE
Guild > place the guildstone

### ISSUE THREAD

**Issue by andrtechno** [polserver/ModernDistro#97]
guild.inc
```
//my add
print(house.IsOwner( mobile )) // return error{ errortext = "Method 'isowner' not found" }

if( !house.IsOwner( mobile ))
     SendSysMessage( mobile, "This is not in your house!", 3, 68 );
     return 0;
...
```

no stone is placed

**Comment by DavidJSGardner** [comment]
> and notice
> 
> ```
> var guild_stone := CreateItemAtLocation( targ.x, targ.y, targ.z, 0xa390, 1, targ.realm );
> ```
> 
> **itemdesc 0x2a390**

If you can change stuff like this and commit it that'd probably be a big help. I'd change this from:
```
var guild_stone := CreateItemAtLocation( targ.x, targ.y, targ.z, 0xa390, 1, targ.realm );
 ```

To:
```
var guild_stone := CreateItemAtLocation( targ.x, targ.y, targ.z, "GuildStone", 1, targ.realm );
 ```

That'd fix the issue and is objtype neutral, meaning the objtype can change and the script won't care.

**Comment by Admin-Yukiko** [comment]
Issue fixed with commit 1dfd729b1c972005da8689c7786fc1591d52100a
